sierra_adapter/terraform/locals.tf

```diff
@@ -1,15 +1,17 @@
 locals {
-  services_alb_listener_http_arn  = "${data.terraform_remote_state.catalogue_pipeline.services_alb_listener_http_arn}"
-  services_alb_listener_https_arn = "${data.terraform_remote_state.catalogue_pipeline.services_alb_listener_https_arn}"
+  alb_cloudwatch_id      = "${module.sierra_adapter_cluster.alb_cloudwatch_id}"
```

Comment:
I’d shortcut locals at this point; the few characters saved isn’t that much clearer now, and it’s easier to see where it comes from within the same stack.